tests/trainer/test_checkpoint.py

```diff
@@ -476,6 +479,7 @@ def test_checkpoint_with_object_store_logger(
     pytest.param("resnet50_synthetic", marks=pytest.mark.daily),
     pytest.param("gpt2_52m", marks=pytest.mark.daily),
 ])
+@pytest.mark.parametrize("different_names", [pytest.param(True), pytest.param(False)])
```

Comment:
Can this be its own test? Not a fan of expanding this parameterization matrix by 2x for this.